guardian/newsletters-nx
# brr email content and form page updates

## What does this change?

adds functions to generate derived properties of newsletters:
 - the "subscribe" and "unsubscribe" event names identity witll use in Brazr
 - the URL for the frontend - rendered sign up form
 - the 'latest in series' endpoint from email rendering for the newsletter's series tag

Includes the above in Email template sent to BRR

Adds the values to the newsletter details pages for reference

removes unnecessary edit fields from the BrazeEditor view of the edit newsletter for and move the 'status' field to the bottom of the page.

## How to test
 - Set yourself to receive the Braze emails when running locally,
 - set the local user profile to "braze editor"
 - run locally
 - launch a newsletter to see the revised email
 - view the details page and edit page

## How can we measure success?

Email has more useful information

## Have we considered potential risks?



## Images

<img width="901" alt="Screenshot 2023-09-25 at 14 47 23" src="https://github.com/guardian/newsletters-nx/assets/30567854/01ac74e0-286f-418d-8dbc-d06d785caf1a">


<img width="901" alt="Screenshot 2023-09-25 at 14 52 31" src="https://github.com/guardian/newsletters-nx/assets/30567854/b54e9577-33d4-46c3-accf-0e730cc1372b">
